### ISSUE TITLE
Fails to build on 32bit platforms

### ISSUE THREAD

**Issue by jbeich** [emersion/grim#37]
Found on FreeBSD [i386](https://ptpb.pw/l9hB) and [armv7](https://ptpb.pw/NkRm):

```c
../cairo_jpg.c:85:42: error: comparison of integers of different signs: 'ssize_t' (aka 'int') and 'unsigned int' [-Werror,-Wsign-compare]
        if (write((long) closure, data, length) < length) {
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^ ~~~~~~
```

**Comment by emersion** [comment]
Can you please send a patch? I guess casting `length` is necessary.

**Comment by ammgws** [comment]
This was fixed in #54 (merged)

**Comment by jbeich** [comment]
[Works](https://github.com/emersion/grim/files/3969362/grim-1.2.0_1.log) for me.
